### ISSUE TITLE
How to process arrays of queries/mutations

### ISSUE THREAD

**Issue by n1ru4l** [Folkloreatelier/laravel-graphql#77]
Is this supported? 

ATM I am doing it like this in my GraphQLController:
Edit:
```php

if( is_array($data) ) {
    $res = [];
    foreach ($data as $_data) {
        $result = app('graphql')->query($_data['query'], $_data['variables'], [
            'context' => $context,
            'schema' => $schema,
            'operationName' => $_data['operationName']
        ]);

        $res[] = $result;
    }

    return $res;
} 
```

**Comment by n1ru4l** [comment]
I added a PR with tests for this!

**Comment by dmongeau** [comment]
Merged in v1.0.16!
